app/package.json

```diff
@@ -30,6 +30,7 @@
     "cardano-crypto.js": "^6.1.0",
     "chacha": "^2.1.0",
     "lodash": "^4.17.21",
+    "moment": "^2.29.1",
```

Comment:
the update of cypress (technically not needed but there shouldn't be any harm in doing so) apparently dropped the 'moment' recursive dependency and we weren't importing it explicitly

Reply:
I directly added a commit t enable "no-extraneous-dependencies" rule and fixed other packages (babel-polyfill and glob) that were apparently also implicitly imported